CRM/Upgrade/Incremental/php/FiveNine.php

```diff
@@ -61,7 +61,7 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
         2 => ts('Email on Hold'),
       );
       $postUpgradeMessage .= '<p>' . ts('If the setting "%1" is enabled, you should update any smart groups based on the "%2" field.', $args) . '</p>' .
-        '<p>' . ts('If you were previously on version 5.8 and altered the WYSIWYG editor setting, you should visit the <a %1>Display Preferences</a> page and re-save the WYSIWYG editor setting.', array(1 => 'href="' . CRM_Utils_System::url('civicrm/admin/setting/preferences/display', 'reset=1') . '"')) . '</p>';
+        '<p>' . ts('If you were previously on version 5.8 and altered the WYSIWYG editor setting, you should visit the <a %1>Display Preferences</a> page and re-save the WYSIWYG editor setting.', array(1 => 'href="' . CRM_Utils_System::url('civicrm/admin/setting/preferences/display', 'reset=1') . '"')) . '</p><p>' . ts('If you have any custom fields of type Money, Int or Float, The search by range checkbox has been unchecked for these fields. If you wish to search by range on those fields you must re-set the checkbox') . '</p>';
```

Comment:
> If you have any custom fields of type Money, Int or Float, The search by range checkbox has been unchecked for these fields. If you wish to search by range on those fields you must re-set the checkbox.

@seamuslee001 When I first read this text, I was pretty alarmed, so then I read more of the the background discussion -- and maybe it was false alarm and just a wording issue? Restating the situation to check my comprehension: This does not apply to *all* `Money|Int|Float` fields -- rather, it specifically applies to `Radio|Select`. To understand why this was done, it helps to consider the visual from the "New Custom Field" screen:

![image](https://user-images.githubusercontent.com/1336047/50618055-4914bf80-0ea5-11e9-9cba-ae6d023b3a4b.png)

In older versions (<=5.8), for fields with `(Money|Int|Float)`+`(Radio|Select)`, you would see the first option but not the second. In v5.9, you would see both. Consequently:

1. Pre-existing fields (from <=5.8) don't have a sensible value of `is_search_range`.
2. Without the patch, pre-existing fields (from <=5.8) effectively change their behavior.
3. With the patch, pre-existing fields (from <=5.8) effectively keep their behavior.
4. If you were using an interim version (5.9-alpha or 5.9-beta?) , then the upgrade script may override the policy -- these are the people who need to update the setting.

How about this language:

> CiviCRM v5.9+ adds a new search preference for certain custom-fields ("Money", "Integer", or "Float" data displayed as "Select" or "Radio" fields). For continuity, any old fields have been set to continue the old user-experience. However, you may want to review these settings. (You should especially review if this site used v5.9-alpha or v5.9-beta.)

Reply:
sure happy with that @totten